snuba/query/snql/__init__.py

```diff
@@ -0,0 +1,28 @@
+from snuba.datasets.dataset import Dataset
+from snuba.query.logical import Query
+from parsimonious.grammar import Grammar
```

Comment:
please sort the imports according to PEP 8. 
Which means external libraries (parsimonious go first)